rust/private/rustc.bzl

```diff
@@ -341,12 +340,16 @@ def construct_arguments(
         out_dir (str): The path to the output directory for the target Crate.
         build_env_files (list): Files containing rustc environment variables, for instance from `cargo_build_script` actions.
         build_flags_files (list): The output files of a `cargo_build_script` actions containing rustc build flags
-        maker_path (File): An optional clippy marker file
         emit (list): Values for the --emit flag to rustc.
 
     Returns:
         tuple: A tuple of the following items
-            - (Args): An Args object with common Rust flags
+            - (struct): A struct of arguments used to run the `Rustc` action
+                - process_wrapper (Args): Arguments for the process wrapper
+                - launcher (Args): Arguments for invoking rustc via the process wrapper
+                - rustc (Args): Rust flags for the Rust compiler
+                - all (list): A list of all `Args` objects in the order listed above.
+                    This is to be passed to the `arguments` parameter of actions
```

Comment:
I wonder if there is naming that will show that the first and the third args contain command line flags, and launcher contains pretty much a path to the rustc binary?

What do you think about:

* process_wrapper_flags (or _opts, probably not _args as it is loaded term)
* rustc_path (launcher is also a bit loaded term, in java/python rules the launcher is a binary that starts the runtime and bundles all the code in a single file)
* rustc_flags (or _opts)

Reply:
Updated 😄 

I, kinda arbitrarily, went with `_flags` but I wonder if there could there be a vocabulary/language guide made for Bazel so decisions like this are clearer across various rules?